### DISCUSSION TITLE
Usage of S3 storage

### DISCUSSION THREAD

**Discussion by w1kke** [humanprotocol/human-protocol#556] [Q&A] [answered]
It does not specify in the SDK readme about the storage whether the S3 bucket has to be publicly accessible. But if I check where and who has to use it, I see that Recording Oracle and Reputation Oracle as well as the Job contributors have to work with the data. This would require them to have access to the data in the S3 bucket.

So my question is, how should the S3 bucket be configured to be used correctly by the whole chain of participants?

**Comment by portuu3** [comment, marked as answer]
You are right, buckets should be public. In a decentralized system all actors should be able to access to the data, but this doesn't mean they can read it. The data might be encrypted but public
